docs/standard/choosing-core-framework-server.md

```diff
@@ -8,8 +8,8 @@ ms.date: 04/20/2021
 
 There are two supported .NET implementations for building server-side apps:
 
-* *.NET Framework*
 * *.NET Core/5+*, which includes .NET Core, .NET 5, and later versions.
+* *.NET Framework*, legacy versions.
```

Comment:
Reversing the order and clarifying versions here makes sense, but "legacy" has connotations that don't apply to .NET Framework.

```suggestion
* *.NET Core/5+*, which includes .NET Core, .NET 5, and later versions.
* *.NET Framework*, which includes 4.8 and earlier versions.
```